cnnlabs/cnn-antools-factors-fe
# alert-item: add alert item component

Adding alert items module.

Fixes: CNNETANTWO-121
Reviewed-By: Katie Owen <katherine.owen@turner.com>

To Test Locally:
1. `git fetch -p`
2. `git checkout CNNETANTWO-121`
3. `git pull` (if prompted)
4. `npm start`
5. User should see a list of alerts.
6. User should be able to click the share URL and paste that text anywhere he/she would like. 

![121](https://user-images.githubusercontent.com/15681479/29527845-11351cd4-8668-11e7-8ddc-0dd46c4b25f1.png)
